### DISCUSSION TITLE
Logs are missing timestamp

### DISCUSSION THREAD

**Discussion by ogmueller** [PremoWeb/chadburn#81] [General]
It would be very helpful, if chadburn logs would include a timestamp in order to see, when an event happened.

currently you only see something like:
```
chadburn                | cron_utils.go:13 ▶ DEBUG removed
chadburn                | cron_utils.go:13 ▶ DEBUG removed
chadburn                | scheduler.go:94 ▶ NOTICE Job deregistered (will not fire again) "optimize" - "php -dmemory_limit=-1 /web/app/console.php index:optimize" - "@every 1d" - ID: 0
chadburn                | scheduler.go:94 ▶ NOTICE Job deregistered (will not fire again) "rebuild" - "php -dmemory_limit=-1 /web/app/console.php index:rebuild" - "@every 2d" - ID: 0
chadburn                | cron_utils.go:13 ▶ DEBUG removed
chadburn                | cron_utils.go:13 ▶ DEBUG removed
```

**Comment by ogmueller** [comment]
Honestly, a rework on the log output would be nice. Maybe a JSON output (at least as option) could be helpful, like many other docker containers (e.g. mattermost, gitlab, ...).

An example could be:
```json
{"timestamp":"2023-11-09 11:28:26.615 +01:00","level":"error","msg":"something went wrong","caller":"scheduler.go:94","container":"traefik"}
```

**Comment by maietta** [comment]
I agree. An accurate timestamp as part of the log is essential. I also agree that we should at least have a JSON representation of the logs as an option.

I might tackle this and a few other small things in the next couple days or so.
